ods_ci/tests/Resources/Page/ModelRegistry/ModelRegistry.resource

```diff
@@ -0,0 +1,400 @@
+    ...        shell=true    stderr=STDOUT
+    Log To Console    ${result.stdout}
+
+Get Domain
+    [Documentation]  Gets the Domain and returns it to 'Get Cluster Domain And Token'.
+    # Run the command to get the ingress domain
+    ${domain_result}=    Run Process    oc    get    ingresses.config/cluster
+    ...    -o    yaml    stdout=PIPE    stderr=PIPE
+    ${rc}=    Set Variable    ${domain_result.rc}
+    IF    $rc > 0    Fail    Command 'oc whoami -t' returned non-zero exit code: ${rc}
+    ${domain_yaml_output}=    Set Variable    ${domain_result.stdout}
+
+    # Return the domain from stdout
+    ${domain_parsed_yaml}=    Evaluate    yaml.load('''${domain_yaml_output}''', Loader=yaml.FullLoader)
+    ${ingress_domain}=    Set Variable    ${domain_parsed_yaml['spec']['domain']}
+
+    # Return both results
+    RETURN    ${ingress_domain}
+
+Get Token
+    [Documentation]    Gets the Token and returns it to 'Get Cluster Domain And Token'.
+    ${token_result}=    Run Process    oc    whoami    -t    stdout=YES
+    ${rc}=    Set Variable    ${token_result.rc}
+    IF    ${rc} > 0    Fail    Command 'oc whoami -t' returned non-zero exit code: ${rc}
+    ${token}=    Set Variable    ${token_result.stdout}
+    RETURN    ${token}
+
+Apply ServiceMeshMember Configuration
+    [Documentation]    Apply a ServiceMeshMember configuration using oc.
+    Copy File    source=${SERVICE_MESH_MEMBER}    destination=${MODELREGISTRY_BASE_FOLDER}/serviceMeshMember.yaml
+    Run    sed -i '' -e 's/<MODEL_REGISTRY_NS>/${NAMESPACE_MODEL_REGISTRY}/' ${MODELREGISTRY_BASE_FOLDER}/serviceMeshMember.yaml    # robocop: disable
+    Apply OpenShift Configuration    ${MODELREGISTRY_BASE_FOLDER}/serviceMeshMember.yaml
+    Remove File    ${MODELREGISTRY_BASE_FOLDER}/serviceMeshMember.yaml
+
+Apply Rest API Configuration
+    [Documentation]    Apply a Rest API configuration using oc.
+    Apply OpenShift Configuration    ${ENABLE_REST_API}
+
+Log File Content
+    [Documentation]    Logs the contents of given file
+    [Arguments]    ${file_path}
+    ${content}=    Get File    ${file_path}
+    Log    ${content}
+
+Append Key Value To Env File
+    [Documentation]    Applies key and value to an env file
+    [Arguments]    ${env_file}    ${key}    ${value}
+    ${formatted_line}=    Set Variable    \n${key}=${value}
+    Append To File    ${env_file}    ${formatted_line}
+
+Run Update Notebook Script
+    [Documentation]    Update the notebook
+    ${result}=    Run Process    python
+    ...    ${IPYNB_UPDATE_SCRIPT}    stdout=TRUE    stderr=TRUE
+    Log    ${result.stdout}
+    Log    ${result.stderr}
+    Should Contain    ${result.stdout}    Modified notebook saved
+
+Remove Model Registry
+    [Documentation]    Run multiple oc delete commands to remove model registry components
+    # We don't want to stop the teardown if any of these resources are not found
```

Comment:
the Run And Verify Command kw fails if the the command fails. 
In theory if this runs as teardown, it shouldn't matter